test/helpers/helpers.go

```diff
@@ -40,6 +41,19 @@ const (
 	// TestOperatorSourceLabelValue is a label value added to the opeator source returned
 	// by the CreateOperatorSource function.
 	TestOperatorSourceLabelValue string = "Community"
+
+	// TestDatastoreCscName is the name of a CatalogSourceConfig that is returned by
+	// the CreateDatastoreCsc function.
+	TestDatastoreCscName string = "test-operators"
```

Comment:
Won't there be a conflict with the child resources of the test `OperatorSource` given they both have the same name?

Reply:
I think you are correct - good catch!

Reply:
I don't think this is an issue though since the goal of this csc is to be correctly referenced by the subscription after upgrade. It isn't deleted or updated or anything that would cause conflicts with the `operatorsource`. Also this is probably intended to mimic the 4.1 scene of having csc's and opsrc's with the same name before upgrade.